openvdb_houdini/houdini/ParmFactory.h

```diff
@@ -338,6 +338,12 @@ class OPENVDB_HOUDINI_API ParmFactory
     /// @details (The default vector size is one element.)
     ParmFactory& setVectorSize(int);
 
+    /// @brief Mark this parameter as hidden from the UI.
+    /// @note Marking parameters as obsolete is preferable to making them invisible as changing
+    /// invisible parameter values will still trigger a re-cook, however this is not possible
+    /// when using multi-parms.
+    ParmFactory& setInvisible(bool);
```

Comment:
Drop the boolean argument.  No one is going to need to `setInvisible(false)`, since that's the default setting.  And double negatives like that are awkward.

Reply:
Agreed, just made that change, thanks.